### PR TITLE
Add a new API endpoint listing the repositories

### PR DESCRIPTION
**Pull request by pypingou**
This endpoint can be used to check the status of mirrormanager's UI

Fixes https://github.com/fedora-infra/mirrormanager2/issues/83

### CODE REVIEW

File: mirrormanager2/api.py [hunk 1]
```diff
@@ -95,3 +95,63 @@ def api_mirroradmins():
...
+          }
+        ]
+        "total": 2
+      }
+
+    '''
+
+    repositories = mmlib.get_repositories(SESSION)
+    repos = []
+    for repositorie in repositories:
```
Comment (ralphbean) [inline mirrormanager2/api.py]: repository instead of repositorie.

File: mirrormanager2/api.py [hunk 2]
```diff
@@ -95,3 +95,63 @@ def api_mirroradmins():
     jsonout = flask.jsonify(output)
     jsonout.status_code = 200
     return jsonout
+
+
+@APP.route('/api/repositories/')
+@APP.route('/api/repositories')
```
Comment (oddshocks) [inline mirrormanager2/api.py]: Is something like this always required? Or only in some cases? (just curious, never had to do this)

**Comment by oddshocks** [comment]
Looks proper to me, homedog! :+1:

File: mirrormanager2/api.py (hunk 2 above)
Comment (pypingou) [inline mirrormanager2/api.py]: It's not required per say, it's just the way to ask flask to support both URL, with and without the trailing slash. After it's a matter of personal preferences if you use it or not :)

**Pull request merged**